test/account.c

```diff
@@ -80,73 +80,70 @@ int test_account(void)
 int test_account_uri_complete(void)
 {
 	static const struct test {
-		const char *in;
-		const char *out;
+		struct pl in;
+		struct pl out;
 	} testv[] = {
 
-		{ "192.168.1.2",
-		  "sip:192.168.1.2" },
+		{ PL("192.168.1.2"),
+		  PL("sip:192.168.1.2") },
```

Comment:
can you please rewrite this to make minimal changes to the test code ?

Reply:
I could leave wrapper function with the current parameter list for `account_uri_complete()` and `uag_find_requri()`. Then
- let test code unchanged
- let contacts and gtk unchanged
- menu will make use of new function and become simplified